stoplight.js

```diff
@@ -0,0 +1,13 @@
+// console.log("stoplight says hello");
+$(document).ready(function(){
+  $("#red").on("click", function(){
+    $(this).toggleClass("active");
+  })
+  $("#yellow").on("click", function(){
+    $(this).toggleClass("active");
+  })
+  $("#green").on("click", function(){
+    $(this).toggleClass("active");
+  })
+
```

Comment:
I figured it out after pushing, instead of calling each light by its ID you would call the .bulb class and toggle the .active class